### DISCUSSION TITLE
Resize the cells for tables

### DISCUSSION THREAD

**Discussion by tio-trom** [standardnotes/forum#2561] [Suggestions]
I am using the Plus Editor and I cannot resize the table cells at all. Is that possible somehow? Thanks!

**Comment by JaspalSuri** [comment]
Hi @tio-trom, there currently isn't a way to do this through the [GUI](https://en.wikipedia.org/wiki/Graphical_user_interface); you would have to use the Code View button and insert styling rules within the `<td>` tags of the table. [Here's an example.](https://stackoverflow.com/questions/7613541/adjusting-table-cell-width)

**Comment by tio-trom** [comment]
I see, Could be a very useful thing to be able to do it via GUI.
